lib/galaxy/config/script.py

```diff
@@ -154,7 +154,8 @@ def _handle_galaxy_yml(args, config_dir, data_dir):
 
     galaxy_config_template = []
     with open(GALAXY_CONFIG_TEMPLATE_FILE) as fh:
-        for line in [l.rstrip('\n') for l in fh.readlines()]:
+        for line in fh.readlines():
```

Comment:
Can we remove readlines here? Maybe a task a another PR?

Reply:
Done, thanks!